src/webextension/list/components/item-summary.css

```diff
@@ -3,23 +3,19 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 .item-summary {
-  padding: 0.5em;
+  padding: 1em 1.3em;
   font-size: 15px;
 }
 
 .item-summary::before {
-  background-image: url(/icons/arrowhead-right-16.svg);
+  background-image: url(/icons/chevron-right.svg);
```

Comment:
Since `icons/arrowhead-right-16.svg` is no longer used, let's remove it entirely.